src/aof.c

```diff
@@ -1376,7 +1376,7 @@ void backgroundRewriteDoneHandler(int exitcode, int bysignal) {
          *
          * There are two possible scenarios:
          *
-         * 1) AOF is DISABLED and this was a one time rewrite. The temporary
+         * 1) AOF is DISABLED and this was one time rewrite. The temporary
```

Comment:
```suggestion
         * 1) AOF is DISABLED and this was a one time rewrite. The temporary
```
i think this change is actually wrong.